### PR TITLE
Lourdes/squarelotron

### PR DESCRIPTION
**Pull request by lourdesVillca**

### CODE REVIEW

File: src/main/java/org/minions/devfund/lourdes/Squarelotron.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import java.util.Arrays;
+
+/**
+ * Class that consists basically of a matrix of numbers.
+ * This matrix can be decomposed as square rings which can flip independently in two different ways:
+ * Upside-Down and through the Main Diagonal.
+ */
+public class Squarelotron {
+    private int size;
+    private int[][] squarelotron;
```
Comment (carledriss) [inline src/main/java/org/minions/devfund/lourdes/Squarelotron.java]: MAJOR Rename field "squarelotron" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1700)

**Comment by carledriss** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by lourdesVillca** [comment]
progress2

**Comment by carledriss** [comment]
@lourdesVillca 
Increase the code coverage.
https://codecov.io/gh/Minions-F/devfund/pull/30/diff?src=pr&el=tree#diff-c3JjL21haW4vamF2YS9vcmcvbWluaW9ucy9kZXZmdW5kL2xvdXJkZXMvU3F1YXJlbG90cm9uLmphdmE=

It is missing unit tests when rotate is sending numberOfTurns negative

**Review by carledriss — CHANGES_REQUESTED** [review]
See the observations

**Review by carledriss — APPROVED** [review]
